### PR TITLE
Fix compilation on ios and provide NopClipboardContext on it

### PR DESCRIPTION
**Pull request by kchibisov**
Fixes: #1

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Changelog
+All notable changes to this project will be documented in this file.
+
+The format is based on [Keep a Changelog](https://keepachangelog.com/en/1.0.0/),
+and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0.html).
+
```
Comment (chrisduerr) [inline CHANGELOG.md]:
```suggestion
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+
+## Unreleased
+
+### Fixed
+
+- Fix compilation on ios and provide NopClipboardContext on it
```
Comment (chrisduerr) [inline CHANGELOG.md]:
```suggestion
- Compilation on iOS, using the no-op clipboard
```

File: Cargo.toml [hunk 3]
```diff
@@ -15,11 +15,11 @@ objc = "0.2"
 objc_id = "0.1"
 objc-foundation = "0.1"
 
-[target.'cfg(all(unix, not(any(target_os="macos", target_os="android", target_os="emscripten"))))'.dependencies]
+[target.'cfg(all(unix, not(any(target_os="macos", target_os="android", target_os="ios", target_os="emscripten"))))'.dependencies]
```
Comment (chrisduerr) [inline Cargo.toml]: All these annotations seem absolutely massive.

I feel like we could simplify them a lot, by just making it either linux or bsd.

Comment (kchibisov) [reply Cargo.toml]: There're a lot of bsd and some solaris, etc. I don't think that we can do anything about it.

Comment (kchibisov) [reply Cargo.toml]: For more information look there https://github.com/rust-windowing/winit/blob/master/Cargo.toml

Comment (chrisduerr) [reply Cargo.toml]: Hm, I guess that's a fair point. Well it's not like that's going to break anything, so it should be fine. Let's just keep this change simple.

**Comment by dvc94ch** [comment]
That fixed the build, thanks! Could you release a new version? :)

**Comment by chrisduerr** [comment]
Releasing a new version makes sense, that should also update all the links to point to the new repo, which is nice. Will bump version in a second!

**Comment by chrisduerr** [comment]
@dvc94ch Should be published as 0.6.2. Let me know if that doesn't work for you.

**Pull request merged**